packages/graphql/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: graphql
 description: A stand-alone GraphQL client for Dart, bringing all the features from
   a modern GraphQL client to one easy to use package.
-version: 1.0.2-beta.2
+version: 1.1.0
```

Comment:
Can you please change this back?

Reply:
ok, sorry i did't pay attention to this.

Reply:
ah no, I meant to merge this to `master`, not `beta`. It seems that I can't change the destination, so I will close this and make another PR.